src/torrent.cpp

```diff
@@ -8283,7 +8277,7 @@ namespace libtorrent
 
 		int ret = 0;
 
-		std::int64_t const fin_time = finished_time();
+		std::int64_t const fin_time = duration_cast<seconds>(finished_time()).count();
```

Comment:
I think you should preserve the type safe representation of std::chrono::seconds (or any duration type) for as long as possible here. no matter what duration type this returns, you can still subtract it from the duration type ``active_time()`` returns.